src/main/groovy/io/seqera/wave/tower/User.groovy

```diff
@@ -44,4 +44,6 @@ class User {
     @Size(max = 255)
     String email
 
+    @Size(max = 100)
+    String organization
```

Comment:
Good try, but this is not the Platform organization but organization name taken from Github when the user sign-in with GH SSO. therefore it's useless. Currently the tower endpoint is the only way to segregate user by org

Reply:
this is also the value user set in its Seqera platform profile

Reply:
Exactly, this is why it's useless

Reply:
ok understood

Reply:
we already have towerEndpoint in DB
Should I create a metric API for that?

Reply:
Let put the organization problem on hold for now. If you could start looking the Conda Fatcher PR I've drafted yesterday it could be great

Reply:
sure, I will start working on it